src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1211,11 +1212,7 @@ export class ResultsViewPageStore {
         ],
         invoke: () => {
             // we get mutations with mutations endpoint, all other alterations with this one, so filter out mutation genetic profile
-            const profilesWithoutMutationProfile = _.filter(
-                this.selectedMolecularProfiles.result,
-                (profile: MolecularProfile) =>
-                    profile.molecularAlterationType !== 'MUTATION_EXTENDED'
-            );
+            const profilesWithoutMutationProfile = getProfilesWithoutMutationProfile(this.selectedMolecularProfiles.result || []);
```

Comment:
probably gauranteed that result will be array here.   i know ts complains so I think trailing ! is sufficient.